analyticscloud/importers/db.py

```diff
@@ -23,6 +23,9 @@
     sqltypes.REAL, sqltypes.SMALLINT
 )
 
+SQL_FLOAT_TYPES = (sqltypes.NUMERIC, sqltypes.Numeric, sqltypes.DECIMAL,
```

Comment:
It looks like Camel names are the sqlachemny names and UPPERCASE are the official SQL names.  Unless, we've seen both in the wild I'd prefer if we only used one or the other.